tmt.spec

```diff
@@ -1,234 +1,149 @@
-Name: tmt
-Version: 1.27.0
-Release: 1%{?dist}
-
-Summary: Test Management Tool
-License: MIT
-BuildArch: noarch
-
-# Build only on arches where libguestfs (needed by testcloud) is available
-%{?kernel_arches:ExclusiveArch: %{kernel_arches} noarch}
-%if 0%{?rhel} >= 9
-ExcludeArch: %{power64}
-%endif
+Name:           tmt
+Version:        1.27.0.dev0
+Release:        %autorelease
+Summary:        Test Management Tool
+
+License:        MIT
+URL:            https://github.com/teemtee/tmt
+Source:         %{url}/releases/download/%{version}/tmt-%{version}.tar.gz
+
+BuildArch:      noarch
+BuildRequires:  python3-devel
 
-URL: https://github.com/teemtee/tmt
-Source0: https://github.com/teemtee/tmt/releases/download/%{version}/tmt-%{version}.tar.gz
+Requires:       git-core rsync sshpass
+
+Obsoletes:      python3-tmt < 1.28
+Obsoletes:      tmt-all < 1.28
```

Comment:
Main package should not obsolete `tmt-all`, right?

Reply:
nice catch, thanks.